src/library/CMakeLists.txt

```diff
@@ -26,47 +26,83 @@
 #  POSSIBILITY OF SUCH DAMAGE.
 #
 
-add_library(commissioner STATIC
-    openthread/bloom_filter.cpp
-    openthread/crc16.cpp
-    openthread/pbkdf2_cmac.cpp
-    openthread/random.cpp
-    openthread/sha256.cpp
-    cbor.cpp
-    coap.cpp
-    commissioner_impl.cpp
-    commissioner_safe.cpp
-    commissioning_session.cpp
-    cose.cpp
-    dtls.cpp
-    error.cpp
-    logging.cpp
-    multicast_dns.cpp
-    network_data.cpp
-    socket.cpp
-    tlv.cpp
-    token_manager.cpp
-    udp_proxy.cpp)
-
-target_link_libraries(commissioner PRIVATE
-    cose
-    mdns
-    mbedtls
-    pthread
-    fmt::fmt
-    event_core_static
-    event_pthreads_static
-    commissioner-common)
+add_library(commissioner)
+target_sources(
+  commissioner
+  PRIVATE openthread/bloom_filter.cpp
+          openthread/bloom_filter.hpp
```

Comment:
Is it necessary to add header files here?

Reply:
yes, otherwise qtcreator and vscode do not show the header files as part of the cmake project...